build/bazelutil/bazel-generate.sh

```diff
@@ -53,6 +53,19 @@ fi
 
 bazel run //:gazelle
 
+if files_unchanged_from_upstream $(find ./pkg -name '*.proto'); then
+  echo "Skipping generation of protobuf dependencies."
+else
+   bazel query 'kind(go_proto_library, //pkg/...)' \
+       | sort \
+       | awk '
+BEGIN {printf("# Generated by bazel-generate.sh\n\nprotobuf_srcs = [\n") }
```

Comment:
Nice!

Reply:
Err, I decided to take this one step further because I think I'm going to generate a bunch of these.